docs/integrations/observability/_index.rst

```diff
@@ -193,4 +193,80 @@ Monitoring Dashboard
    kubectl -n det-monitoring port-forward svc/monitor-grafana 9000:80
 
 -  Navigate to `127.0.0.1:9000 <http://127.0.0.1:9000>`__. Sign in with the ƒusername ``admin`` and
-   the password you set above. You should see the ``Determined API Server Monitoring`` dashboard.
+   the password you set above. You should visit the ``Determined API Server Monitoring`` dashboard.
+
+******************
+ Metrics
+******************
+Determined does not produce metrics, rather it uses existing tools to report information. Visit 
+source documentation for details on specific metrics.
```

Comment:
what is the URL for the source documentation that they should visit?

Reply:
The source documentation links are listed below. Would it make more sense to say "Visit source documentation listed below for details on specific metrics"?

Reply:
ok makes sense. let's remove the sentence else they'll go looking for something that's already obvious enough.

******************
 Metrics
******************
Determined does not produce metrics, rather it uses existing tools to report information.